### PR TITLE
added date input - that can take any date as value and radio chip true or false inputs works as expected 

### PR DESCRIPTION
**Pull request by GSuma21**

### CODE REVIEW

File: projects/dynamic-form/src/lib/components/main-form/main-form.component.html [hunk 1]
```diff
@@ -132,7 +132,7 @@
 
     <mat-form-field appearance="outline" *ngIf="control?.type === 'radio'" floatLabel="always">
       <mat-label *ngIf="control.label">{{ control?.label }}</mat-label>
-      <input matInput [formControlName]="control?.name" hidden />
+      <input matInput [formControlName]="control?.name" hidden style="visibility: hidden; display:none;" />
```
Comment (ramkumar-ns-dev) [inline projects/dynamic-form/src/lib/components/main-form/main-form.component.html]: please move this to styles file

File: projects/dynamic-form/src/lib/components/main-form/main-form.component.html [hunk 2]
```diff
@@ -92,8 +92,8 @@
         matInput
         [formControlName]="control?.name"
         hidden
+        style="visibility: hidden; display:none;"
```
Comment (ramkumar-ns-dev) [inline projects/dynamic-form/src/lib/components/main-form/main-form.component.html]: please move this to styles file

File: projects/dynamic-form/src/lib/components/main-form/main-form.component.html [hunk 3]
```diff
@@ -152,6 +152,7 @@
...
```
Comment (ramkumar-ns-dev) [inline projects/dynamic-form/src/lib/components/main-form/main-form.component.html]: here as well

**Review by ramkumar-ns-dev — CHANGES_REQUESTED** [review]

**Comment by GSuma21** [comment]
fixed please check.

**Pull request merged**